src/frontend/components/ViewersList/index.spec.tsx

```diff
@@ -291,4 +321,25 @@ describe('<ViewersList /> when user is a student', () => {
     );
     screen.getByText('Student 2');
   });
+
+  it('displays number of anonymous when there is some', () => {
+    const video = videoMockFactory();
+
+    useParticipantsStore.setState({
+      participants: [
+        { ...participant1, name: 'anonymous-XXX' },
+        { ...participant2, name: 'anonymous-YYY' },
+        { ...participant3, name: 'anonymous-ZZZ' },
+      ],
+    });
+
+    render(
+      wrapInIntlProvider(<ViewersList isInstructor={false} video={video} />),
+    );
+    expect(screen.queryByText('Other participants')).toBeInTheDocument();
+    expect(
+      screen.queryByText('No viewers are currently connected to your stream.'),
+    ).not.toBeInTheDocument();
+    screen.getByText('2 anonymous viewers.');
```

Comment:
should not be 3 anonymous viewers ?

Reply:
It's because `participant1` is an instructor, but I agree with you, it's a bit confusing. I changed this test to make it clearer.